Content.Server/Ame/Components/AmeControllerComponent.cs

```diff
@@ -72,10 +72,23 @@ public sealed partial class AmeControllerComponent : SharedAmeControllerComponen
     [DataField("nextUpdate")]
     public TimeSpan NextUpdate = default!;
 
+    /// <summary>
+    /// The next time this will try to update the controller UI.
+    /// </summary>
+    [DataField("nextUIUpdate")]
+    public TimeSpan NextUIUpdate = default!;
```

Comment:
The test failures appear to be related to this:

```
   Failed LoadSaveTicksSaveBagel [51 s]
  Error Message:
     Expected string length 3223500 but was 3223557. Strings differ at index 499944.
  Expected: "...rent: 2\n      type: Transform\n- proto: AnalysisComputerCir..."
  But was:  "...rent: 2\n      type: Transform\n    - nextUIUpdate: 64.06642..."
```

Reply:
I removed the "DataField" line since this will only ever be used internally. The IntegrationTest run on my machine came back without errors, so I guess that fixes it?